src/command-line/uninstall.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+const colors = require("colors/safe");
+const program = require("commander");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("uninstall <package>")
+	.description("Uninstall a theme or a package")
+	.on("--help", Utils.extraHelp)
+	.action(function(packageName) {
+		const fs = require("fs");
+		const child = require("child_process");
+
+		if (!fs.existsSync(Helper.getConfigPath())) {
+			log.error(`${Helper.getConfigPath()} does not exist.`);
+			return;
+		}
+
+		log.info(`Uninstalling ${colors.green(packageName)}...`);
+
+		const packagesParent = Utils.preparePackagesDir();
```

Comment:
I don't think you need to prepare it in uninstall, just bail out if packages folder does not exist.

Reply:
Done. It's overall a wee bit messier than what I would have hoped, but oh well. I've kept my original commit in case you preferred the first version in the end, so if you're happy with this don't merge as is, I'll have to squash the extra commit.